src/components/LoadingSkeletons/SingleLineLoadingSkeleton.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import LoadingMessage from '../LoadingMessage/LoadingMessage';
+import LoadingSkeleton, {
+	ILoadingSkeletonProps,
+	IStandardSkeleton,
+} from './LoadingSkeleton';
+import { cxBackgroundGray } from './LoadingSkeletonsSvgUtil';
+
+export const SingleLineSkeleton = (
+	props: IStandardSkeleton
+): React.ReactElement => {
+	const { width = '100%', height = '100%', className } = props;
+
+	return (
+		<div data-test-id='loadingSkeleton-SingleLineSkeleton'>
+			<svg
+				data-test-id='loadingSkeleton-SingleLineSkeleton-svg'
+				width={width}
+				height={height}
+				xmlns='http://www.w3.org/2000/svg'
+			>
+				<g
+					id='SingleLineSkeleton-Details'
+					stroke='none'
+					strokeWidth='1'
+					fill='none'
+					fillRule='evenodd'
+				>
+					<g id='SingleLineSkeleton-Group'>
+						<g
+							className={cxBackgroundGray('&', className)}
+							id='SingleLineSkeleton-Group-1'
+						>
+							<rect
+								id='SingleLineSkeleton-Rectangle'
+								x='0'
+								y='0'
+								width={width}
+								height={height}
+							/>
+						</g>
+					</g>
+				</g>
+			</svg>
+		</div>
+	);
+};
+
+const SingleLineLoadingSkeleton = (
+	props: ILoadingSkeletonProps
+): React.ReactElement => {
+	return (
+		<LoadingSkeleton
+			data-test-id='loadingSkeleton-SingleLineLoadingSkeleton'
+			Skeleton={SingleLineSkeleton}
+			{...props}
+		/>
+	);
+};
+
+SingleLineLoadingSkeleton.LoadingMessage = LoadingMessage;
```

Comment:
Is `LoadingMessage` necessary? I feel it is a dead code.

Reply:
Good point. It is not necessary.  But since i have merged number of skeletons with this code I would rather remove it as a separate PR. I will be adding "cleanup" PR that will address common skeleton issues that already have been merged
https://jira.xandr-services.com/browse/BWAP-719